PENDING_PENDING_RELEASE_NOTES.md

```diff
@@ -2,6 +2,29 @@
 ## Version 16.2.1
 _Date TBD_
 
+### New Features
+* Add nonconformity charts statistics endpoint
```

Comment:
These top lines are already on PROD. The next few are on STG. I think only line 16 should be in this section. Also, if there are actually any new features then the Version should be different